### PR TITLE
Partial update & update numpy configuration file.

### PR DESCRIPTION
**Pull request by Carreau**
Add option to regenerate documentation only for some items.

    When debugging or iterating on the docs it should be interesting to
    be able to update the intermediate bundle with only some elements
    For example because you are updating the docs.

    This adds a flag --only , that can be passed multiple time at the CLI,
    and will ask the gen step to only collect documentation for those
    objects, and update the doc bundle instead of regenerating it.

    It is likely imperfect and  hence why it's only a cli flag.

    I'm not happy withe the name --only. Maybe --update ?

### CODE REVIEW

File: papyri/gen.py [hunk 1]
```diff
@@ -1832,6 +1854,11 @@ def collect_api_docs(
             )
 
         collected = {k: v for k, v in collected.items() if k not in excluded}
+        if limit_to:
+            collected = {k: v for k, v in collected.items() if k in limit_to}
+            print("DEV: regenerating docs only for")
```
Comment (aktech) [inline papyri/gen.py:1859]: I see the logging is setup already, any specific reason for using `print`?

File: papyri/gen.py [hunk 2]
```diff
@@ -1794,17 +1814,19 @@ def collect_package_metadata(self, root, relative_dir, meta):
...
         """
         Crawl one module and stores resulting docbundle in self.store.
 
         Parameters
         ----------
         root : str
             module name to generate docbundle for.
+        limit_to : list of string
+            For partial documentation building and testing purposes
+            we may want to generate documentation only a single item.
```
Comment (aktech) [inline papyri/gen.py:1827]:
```suggestion
            we may want to generate documentation for only a single item.
```

**Review by aktech — COMMENTED** [review]
Some minor nitpicks, otherwise looks good.

File: papyri/gen.py [hunk 3]
```diff
@@ -1794,17 +1814,19 @@ def collect_package_metadata(self, root, relative_dir, meta):
...
+            If this list is non-empty we will collect documentation
+            just for this item.
```
Comment (aktech) [inline papyri/gen.py:1829]:
```suggestion
            just for these items.
```

File: papyri/gen.py (hunk 1 above)
Comment (Carreau) [reply papyri/gen.py:1859]: Good point.

**Pull request merged**